docs/build/guides/conventions/upgrading-contracts.mdx

```diff
@@ -30,7 +30,7 @@ Upgrading a smart contract allows you to improve or modify your contract without
 
 The [upgradeable contract example] demonstrates how to upgrade a Wasm contract.
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)][oigp]
+[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?repo=stellar/soroban-examples&editor=web)
```

Comment:
is it possible to formulate a codespace link so that a particular file in the repository opens up? like the `/upgradeable_contract/src/lib.rs` file in this instance?

Reply:
It's really surprising but there's not an easy way to do this but there is a proposal in the finalization stage here:  https://github.com/devcontainers/spec/issues/22

Reply:
I am a watcher on that issue so I'll keep an eye on it.